stylesheet/main.css

```diff
@@ -15,12 +17,25 @@ a {
 
 header nav a {
     color: #5E5369;
+    display: block;
+    padding-top: 1em;
+    padding-bottom: 0.5em;
+    margin-left: 1em;
+    margin-right: 1em;
+    border-top: 4px solid transparent;
+}
+header nav a:hover {
+    border-top-color: red;
 }

```

Comment:
This rule makes it so the border color of hyperlinks changes when they are being hovered over